src/keybindings.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Does it use any non-ascii char? If not, let's drop the encoding for coherence with the rest of the files, later we can add to all of them at once.